### ISSUE TITLE
[TF2] cannot simulate norm or beta input data: eta_loc_j  not implemented

### ISSUE THREAD

**Issue by picciama** [theislab/batchglm#95]
If you want to simulate input data with norm or beta noise, it is not possible because the `eta_loc_j` function is not implemented in the respective simulator classes within `batchglm/models/glm_norm/model.py` and `batchglm/models/glm_beta/model.py`
The problem is that the noise mode specific simulator classes inherit the model, so the abstract method must be defined in the model in order to create an instance of simulator.
Result:
```
TypeError: Can't instantiate abstract class Simulator with abstract methods eta_loc_j
```
@davidsebfischer can you please have a look at it and implement the two missing methods?

**Comment by davidsebfischer** [comment]
Will add this, this was added as part of the dask extension which is only available for NB so far.

**Comment by picciama** [comment]
This is fixed after merge of #118 into dev.
